osu.Game.Rulesets.Osu/Difficulty/Evaluators/AimEvaluator.cs

```diff
@@ -102,7 +103,7 @@ public static double EvaluateDifficultyOf(DifficultyHitObject current, bool with
                                   * DifficultyCalculationUtils.Smootherstep(currAngle, double.DegreesToRadians(110), double.DegreesToRadians(60))
                                   * DifficultyCalculationUtils.Smootherstep(osuLastObj.LazyJumpDistance, radius, diameter)
                                   * Math.Pow(DifficultyCalculationUtils.ReverseLerp(osuLastObj.LazyJumpDistance, diameter * 3, diameter), 1.8)
-                                  * DifficultyCalculationUtils.Smootherstep(lastAngle, double.DegreesToRadians(110), double.DegreesToRadians(60));
+                                  * DifficultyCalculationUtils.Smootherstep(Math.Min(lastAngle, lastLastAngle), double.DegreesToRadians(110), double.DegreesToRadians(60));
```

Comment:
That just makes the wiggle bonus potentially completely ignore the second angle no matter if its wide or acute. I don't think it makes any sense to be done like that in a _acute wiggle bonus_

Reply:
this should buff patterns like these without touching anything else
because the main effect of this line is not buffing angle changes on streams, what is still true with this change, because streams have many wide angles in a row
![image](https://github.com/user-attachments/assets/70478088-443f-43c3-b8fb-7d86dd62cffd)